libraries/AP_Stats/AP_Stats.cpp

```diff
@@ -123,13 +124,16 @@ void AP_Stats::update()
 
 void AP_Stats::set_flying(const bool is_flying)
 {
+    static uint32_t oneFlightTime = 0;
     if (is_flying) {
         if (!_flying_ms) {
             _flying_ms = AP_HAL::millis();
+            oneFlightTime = _flying_ms;
         }
     } else {
         update_flighttime();
         _flying_ms = 0;
+        GCS_SEND_TEXT(MAV_SEVERITY_INFO, "current flight time is %.1f sec", (AP_HAL::millis() - oneFlightTime)*0.001f);
```

Comment:
Might get a bit spamy sometimes. Also doesn't make much sense on Rover and Tracker.